include/coap2/net.h

```diff
@@ -716,13 +716,18 @@ unsigned int coap_calc_timeout(coap_session_t *session, unsigned char r);
 /**
  * Function interface for joining a multicast group for listening
  *
- * @param ctx   The current context
+ * @param ctx       The current context
  * @param groupname The name of the group that is to be joined for listening
+ * @param ifname    Index of network interface to join the group on
```

Comment:
`ifname` specifies the name of the  network interface rather than its index. I suggest: s/Index of//.

Reply:
Thanks for catching this.